app/models/agency.rb

```diff
@@ -14,6 +14,10 @@ class Agency < ApplicationRecord
     where(arel_table[:name].lower.matches("%#{text.downcase}%"))
   end
 
+  def self.agency_names
+    distinct.where.not(name: nil).order(:name).pluck(:name, :id).select(&:present?).uniq.map { |v, k| [v, k] }.to_h
+  end
+
```

Comment:
I think this ended up not being used, so maybe we can drop it?

Reply:
correct and thanks for catching that. I'll get it pulled out of the PR.